content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,190 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
+          ➤ IBM Cloud Kubernetes Service (IKS)
+        </td>
+        <td>
+          ➤ <a href="https://github.com/IBM/manifests/tree/{{% iks/latest-branch %}}">Official Website</a>
+          <br>
+          ➤ <a href="/docs/distributions/ibm/">Other Docs</a>
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on Nutanix</td>
         <td>Nutanix</td>
-        <td>Nutanix Karbon</td>
-        <td>{{% nutanix/latest-version %}}</td>
-        <td><a href="/docs/distributions/nutanix/">Docs</a></td>
-        <td></td>
+        <td>
+          ➤ Nutanix Kubernetes Engine
+        </td>
+        <td>
+          ➤ <a href="https://github.com/nutanix/karbon-platform-services/tree/master/automation/infrastructure/terraform/kcs/install_kubeflow">Official Website</a>
+          <br>
+          ➤ <a href="/docs/distributions/nutanix/">Other Docs</a>
+        </td>
       </tr>
       <tr>
-        <td>Kubeflow on OpenShift</td>
+        <td>Open Data Hub</td>
         <td>Red Hat</td>
-        <td>OpenShift</td>
-        <td>1.6</td>
-        <td><a href="/docs/distributions/openshift/">Docs</a></td>
-        <td><a href="https://opendatahub.io/docs/kubeflow.html">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Argoflow</td>
-        <td>Argoflow Community</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.3</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/argoflow/argoflow">External Website</a></td>
+        <td>
+          ➤ OpenShift
+        </td>
+        <td>
+          ➤ <a href="https://opendatahub.io/docs/kubeflow.html">Official Website</a>
+          <br>
+          ➤ <a href="/docs/distributions/openshift/">Other Docs</a>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Kubeflow as a Service</td>
-        <td>Arrikto</td>
-        <td>Fully Managed</td>
-        <td>1.4</td>
-        <td>N/A</td>
-        <td><a href="http://kubeflow.arrikto.com/">External Website</a></td>
+        <td rowspan="2">Arrikto</td>
+        <td>
+          ➤ N/A (Fully Managed)
+        </td>
+        <td>
+          ➤ <a href="https://www.arrikto.com/kubeflow-as-a-service/">Official Website</a>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Enterprise Kubeflow</td>
-        <td>Arrikto</td>
-        <td>EKS,
-            AKS,
-            GKE
-        </td>
-        <td>1.4</td>
         <td>
-          <a href="/docs/distributions/ekf/">Docs</a>
+          ➤ Amazon Elastic Kubernetes Service (EKS)
+          <br>
+          ➤ Azure Kubernetes Service (AKS)
+          <br>
+          ➤ Google Kubernetes Engine (GKE)
         </td>
         <td>
-          <a href="https://www.arrikto.com/enterprise-kubeflow/">External Website</a>
+          ➤ <a href="https://www.arrikto.com/enterprise-kubeflow/">Official Website</a>
         </td>
       </tr>
       <tr>
         <td>Charmed Kubeflow</td>
         <td>Canonical</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.6</td>
```

Comment:
I'm not sure if there was any previous discussion on this. Could you share the reason for removing the latest version supported by the distributions? I feel that this is the most important info on this page for the community to determine which distributions are "up to date." I think it's super valuable to have a page that shows lists of distributions + the latest version they support.

Reply:
@annajung previously I agreed with you, in fact, I made this page initially and was the one to add the version!

However, I now think separating by "active" vs "inactive" is more useful to users (especially new ones).

For most users, this is the first page they see when they want to start using Kubeflow, the goal of this page is simply to show them the options they have to use kubeflow in their environment.

----

When 1.7 is released, we can consider adding a second "active" table, which lists "distributions supporting Kubeflow 1.7 or later". 

However, this may not be necessary, as being "active" should imply they will support new versions in a timely manner.

Reply:
I agree with @annajung here, the version that each distribution supports seems to be one of the most important information the table provides. There also might be distributions who do active maintenance and support of an older version and might not update on time. Letting users know that they might get an older version with those distributions is important.

Reply:
@DomFleischmann Currently, this PR has not reduced the amount of version information, because all the "active" distributions support Kubeflow 1.6, and the heading above the table says `"distributions that provide Kubeflow 1.6 or later"`.

When 1.7 is released, we can consider adding a second "active" table, which lists `"distributions supporting Kubeflow 1.7 or later"`. However, this may not be necessary, as being "active" should imply they will support new versions in a timely manner.

Reply:
Maybe @kimwnasptd can chime in here, but Arrikto currently does not support 1.6+, according to https://www.kubeflow.org/docs/started/installing-kubeflow/. They're still on 1.4, so the changes here wouldn't make sense for Arrikto to be part of the "active" list.

Reply:
@andreeamun we need to confirm with Arrikto, what versions are supported by "Arrikto Kubeflow as a Service" and "Arrikto Enterprise Kubeflow", because I think the table was out of date.

@akgraner can you please confirm?

Reply:
As part of what I changed in https://github.com/kubeflow/website/pull/3381#issuecomment-1309845940, I have now removed any mention of specific Kubeflow versions, so it is less important to confirm the exact "versions" included in Arritko's distributions.